internal/controllers/machinedeployment/machinedeployment_rollout_ondelete.go

```diff
@@ -111,6 +111,7 @@ func (p *rolloutPlanner) reconcileOldMachineSetsOnDelete(ctx context.Context) {
 		scaleDownCount := max(ptr.Deref(oldMS.Spec.Replicas, 0)-ptr.Deref(oldMS.Status.Replicas, 0), 0)
 		if scaleDownCount > 0 {
 			newScaleIntent := max(ptr.Deref(oldMS.Spec.Replicas, 0)-scaleDownCount, 0)
+			p.addNote(oldMS, "scale down unavailable or not existing Machines")
```

Comment:
Is this really unavailable in the usual sense of available? (I couldn't find any occurrence of available around here)

Scale down not existing Machines also sounds a bit strange (if they don't exist why do we have to scale them down? We should probably phrase this slightly differently like the note in l.142)